interface/billing/search_payments.php

```diff
@@ -468,13 +468,13 @@ function SearchPayingEntityAction()
           </tr>
           <tr>
             <td align="right"></td>
-			<td align="left" class="text"><?php echo htmlspecialchars( xl('Payment From'), ENT_QUOTES).':' ?></td>
+			<td align="left" type="text" class="text"><?php echo htmlspecialchars( xl('Payment From'), ENT_QUOTES).':' ?></td>
```

Comment:
That's odd. Since we're here working on the form I suggest we go ahead and use an _actual_ input element instead of a `<td>`